tests/unit/_config/test_compute_options.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import bigframes._config.compute_options as compute_options
+
+
+def test_default_options():
+    options = compute_options.ComputeOptions()
+
+    assert options.allow_large_results is False
```

Comment:
Should be `None` because we need a fallback to the bigquery options if it's not set.